### DISCUSSION TITLE
请帮我核对一下spiderX的写法是否有误

### DISCUSSION THREAD

**Discussion by imtv** [XTLS/Xray-core#1799] [Q&A]
这是一个真实的链接：
https://elements.envato.com/urban-opener-2R5PHEJ

客户端：
"serverName": "elements.envato.com"
"spiderX": "/urban-opener-2R5PHEJ"

请问各位这么写是对的吗？
还是只要随便填一个随机字符串就行

**Comment by lk29** [comment]
是的，我也很感兴趣 
**RPRX** 请回答

**Comment by RPRX** [comment]
其实，上个月我就写好了对 spiderX 参数的说明，~~只是一直没更新 example~~，可能还要再加点内容，~~我是不急的，有空时再更新~~

**Reply by imtv** [reply]
好的，等你更新，爱你😘
